libs/cdk/src/lib/components/picture/picture.component.html

```diff
@@ -3,7 +3,7 @@
     <ng-container *ngFor="let source of sources; trackBy: trackBySrc">
       <source
         *etLet="combineWithConfig(source) as s"
-        [type]="s.type"
+        [type]="(s.srcset | inferMimeType) ?? s.type"
```

Comment:
The user-provided type should have priority over the inferred one

Reply:
"s" is currently { srcset: string, type: string } and thus requiring `type` to be supplied.
It would be nice if "s" could be either that or simply a string representing the srcset